signer/src/network/mod.rs

```diff
@@ -26,6 +37,80 @@ pub trait MessageTransfer {
     fn receive(&mut self) -> impl Future<Output = Result<Msg, Self::Error>> + Send;
 }
 
+/// MessageTransfer interface for the application signalling channel.
+/// TODO: Better name?
+pub struct SignallingNetwork {
```

Comment:
I guess the fact that this uses the signer signals is an implementation detail, but in general this represents a hook to the p2p network, so perhaps it should simply be named `P2PNetwork`.

Reply:
Changed in [376cace](https://github.com/stacks-network/sbtc/pull/473/commits/376cacef68fb40239e51df5049feabaeed3189f6)